azure-simple/packer/Vagrantfile

```diff
@@ -0,0 +1,40 @@
+#----------------------------------------------------------------------------
+#  Copyright (c) 2018 WSO2, Inc. http://www.wso2.org
```

Comment:
@rasanjaya85 the license header is outdated.

Reply:
Done